kedro/runner/parallel_runner.py

```diff
@@ -112,7 +120,11 @@ def _run_node_synchronization(  # pylint: disable=too-many-arguments
         conf_logging = conf_logging or {}
         _bootstrap_subprocess(package_name, conf_logging)
 
-    return run_node(node, catalog, is_async, run_id)
+    hook_manager = create_hook_manager()
+    _register_hooks(hook_manager, settings.HOOKS)
+    _register_hooks_setuptools(hook_manager, settings.DISABLE_HOOKS_FOR_PLUGINS)
```

Comment:
Create a new `hook_manager` when doing multiprocessing, because the `PluginManager` can't be serialised.

Reply:
> `PluginManager` can't be serialised

Is that why we need it in `fork` mode as well? Or can we move it up under the `if` branch?

Also we should also update the docstrings, they still mention "activating the session" which we don't do anymore.

Reply:
Yes, the problems from `PluginManager` not being serialisable happen in all modes, because we try to do the serialisation further up the stack. I tested this by creating a `PluginManager` and calling `pickle.dump()` on it and that fails. 

And yes good point on the docstring!